config/config.exs

```diff
@@ -67,6 +67,22 @@ config :spacesuit, :health_route, %{ path: "/health", enabled: true }
 # Do we call out to an external session service that can process JWT tokens?
 config :spacesuit, session_service: %{ enabled: false }
 
+# Configuration for the CORS middleware
+config :spacesuit, cors: %{
+  # Main kill switch to disable the middleware
+  enabled: false, 
+  # Required prefix match in order to process CORS
+  path_prefixes: ["/matched"],
```

Comment:
This value of `/matched` is necessary for some of the unit tests to succeed and is probably going to be implementation-specific. Do you prefer it here over the test config file? I realize it'll probably be overridden via an environment-specific config file but just want to make sure that putting it here was intentional and desired.